lzma/C/Makefile.am

```diff
@@ -59,5 +59,9 @@ $(ASM_7z).lo: $(ASM_S)
 	$(ASM_PROG) $(ASM_OPT) -o $(ASM_7z).o $(ASM_S)
 	mkdir -p .libs
 	cp $(ASM_7z).o .libs/
-	@echo -e "$(7ZIPASMLOFILE)" > $(ASM_7z).lo
```

Comment:
`echo -e` is not portable. some shell not support this argument.

Reply:
I would not do this. the **.lo** file is needed to build the library in `lrzip 0.641`. Perhaps the `cat` command could be used. Mac users have compiled lrzip successfully. The `echo -e` command allows escape characters to print to the output. Perhaps try hard coding /usr/bin/echo since shells are not guaranteed to work the same way as the bin.